### PR TITLE
feature: webhooks

### PR DESCRIPTION
**Pull request by shwstppr**
Documentation for https://github.com/apache/cloudstack/pull/8674

----
📚 Documentation preview 📚: https://cloudstack-documentation--385.org.readthedocs.build/en/385/

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@shwstppr , if I search for webhooks I get both a hit on the events page and a separate page. Is that on purpose? These seem to contain the same text.

also:
image
I think the text "|create-webhooks.png|"/link should not be there, but the picture should be embedded.

File: source/adminguide/events/webhooks.rst [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      POST requests for event notications must be sent with strict SSL
+      verification request when a HTTPS payload URL is used.
+
+   -  **Secret Key**. An option secret key parameter which can be used to sign
+      the HTTP POST requests for event notifications with HMAC.
+
+   -  **Enabled**. To specify whether the webhook be created with enabled or
+      disabled state
+
+   |create-webhooks.png|
```
Comment (shwstppr) [inline source/adminguide/events/webhooks.rst:93]:
```suggestion
   |create-webhook.png|
```

**Comment by shwstppr** [comment]
@blueorangutan docbuild

@DaanHoogland I'm not sure about the way search works on the documentation website but since webhooks-related notes are in a separate .rst file, it could be the reason two search entries are showing.
I've added a fix for image not showing

**Review by DaanHoogland — APPROVED** [review]

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Review by kiranchavala — APPROVED** [review]
LGTM , the documentation looks good

**Review by rohityadavcloud — APPROVED** [review]
LGTM, let's merge this when the linked PR is merged.

**Pull request merged**